lib/utilities/markdown-color.js

```diff
@@ -33,109 +31,117 @@ module.exports = MarkdownColor;
+    let markdownOptions = (options && options.markdownOptions) || {};
+    const markdown = require('markdown-it');
+    const terminal = require('markdown-it-terminal');
+    this.options = options || {};
+    this.markdown = markdown().use(terminal, markdownOptions);
+    this.tokens = merge(tokens, optionTokens);
   }
 
-  return this.render(file, options);
-};
+  /*
+    Lookup markdown file and render contents
+
+    @private
+    @method renderFile
+    @param {String} [filePath]
+    @param {Object} [options]
+  */
+  renderFile(filePath, options) {
+    let file;
+    try {
+      file = fs.readFileSync(filePath, 'utf-8');
+    } catch (e) {
+      throw new SilentError(`The file '${filePath}' doesn't exist. Please check your filePath`);
+    }
 
-/*
-  Parse markdown and output as string
-  @method render
-  @param {String} [string]
-  @param {Object} [options]
-  @return {String}
-*/
-MarkdownColor.prototype.render = function(string, options) {
-  let indent = (options && options.indent) || '';
-  let input = this.markdown.render(string);
-  let styles = Object.keys(this.tokens);
-  input = input.replace(/^/mg, indent);
-  styles.reverse().map(style => {
-    input = input.replace(this.tokens[style].pattern, this.tokens[style].render);
-  });
-  input = input.replace(/\~\^(.*)\~\^/g, escapeToken);
-  return input;
-};
+    return this.render(file, options);
+  }
 
-/*
-  Generate default style tokens
-  @method generateTokens
-  @return {Object}
-*/
-MarkdownColor.prototype.generateTokens = function(renderer) {
-  let defaultTokens = {
-    // ember-cli styles
-    option: {
-      name: 'option',
-      token: '--option',
-      pattern: /((--\w*\b)|(<options>))/g,
-      render: renderStylesFactory(renderer, 'cyan'),
-    },
-    default: {
-      name: 'default',
-      token: '(Default: default)',
-      pattern: /(\(Default:\s.*\))/g,
-      render: renderStylesFactory(renderer, 'cyan'),
-    },
-    required: {
-      name: 'required',
-      token: '(Required)',
-      pattern: /(\(Required\))/g,
-      render: renderStylesFactory(renderer, 'cyan'),
-    },
-  };
+  /*
+    Parse markdown and output as string
+
+    @method render
+    @param {String} [string]
+    @param {Object} [options]
+    @return {String}
+  */
+  render(string, options) {
+    let indent = (options && options.indent) || '';
+    let input = this.markdown.render(string);
+    let styles = Object.keys(this.tokens);
+
+    input = input.replace(/^/mg, indent);
+    styles.reverse().map(style => {
+      input = input.replace(this.tokens[style].pattern, this.tokens[style].render);
+    });
+    input = input.replace(/\~\^(.*)\~\^/g, escapeToken);
+
+    return input;
+  }
 
-  let colorTokens = unshiftValue(colors.concat(backgroundColors).map(getToken), {}).reduce(setToken);
-  return merge(colorTokens, defaultTokens);
-};
+  /*
+    Generate default style tokens
+
+    @method generateTokens
+    @return {Object}
+  */
+  generateTokens(renderer) {
```

Comment:
it looks like this method is internal to the class; at the very least we should add `@private` to the docs and change the name to `_ generateTokens` imo. I'd probably go as far as extracting this as a util function and not putting it on the class.